src/yetibot/commands/base64.clj

```diff
@@ -0,0 +1,28 @@
+= (ns yetibot.commands.base64
+    (:require
+      [yetibot.core.hooks :refer [cmd-hook]]
+      [clojure.data.codec.base64 :as b64]))
+
+(defn into-bytes
+  "turns a string into a byte array"
+  [s]
+  (byte-array (map (comp byte int) s)))
```

Comment:
btw, another way to do this is `(.getBytes s)`. No strong preference though, just FYI. Can leave this if you like.